src/util/fields.py

```diff
@@ -654,6 +654,19 @@ def convert_field_to_html(cr, model, field, skip_inherit=()):
         """,
         [field, model],
     )
+
+    # Update ir.default
+    if table_exists(cr, "ir_default"):
+        json_value_html = pg_text2html("json_value")
+        query = """
+                UPDATE ir_default AS d
+                   SET json_value = {}
+                  FROM ir_model_fields AS imf
+                 WHERE imf.name = {}
+                   AND imf.model = {}
+                   AND imf.id = d.field_id
+                """
+        format_query(cr, query, sql.Literal(json_value_html), sql.Literal(field), sql.Literal(model))
```

Comment:
You still need to execute the query. And the json_value_html is not a Literal.
```suggestion
        query = """
                UPDATE ir_default AS d
                   SET json_value = {}
                  FROM ir_model_fields AS imf
                 WHERE imf.name = %s
                   AND imf.model = %s
                   AND imf.id = d.field_id
                """
        cr.execute(format_query(cr, query, sql.SQL(json_value_html)), [field, model])
```